src/vdaf/xof.rs

```diff
@@ -446,6 +450,34 @@ impl RngCore for SeedStreamFixedKeyAes128 {
     }
 }
 
+/// XOF based on HMAC-SHA256 and AES128. This XOF is not part of the VDAF spec.
+#[derive(Clone, Debug)]
+pub struct XofHmacSha256Aes128(Hmac<Sha256>);
```

Comment:
This struct and the impl block below need to be guarded with `#[cfg(feature = "crypto-dependencies")]` as well for a `--no-default-features` build to succeed.